src/components/Books.js

```diff
@@ -0,0 +1,54 @@
+const Books = () => {
+  const books = [
+    {
+      id: 1,
+      title: 'The Road to React',
+      author: 'Robin Wieruch',
+    },
+    {
+      id: 2,
+      title: 'Learning Node',
+      author: ' Shelley Powers',
+    },
+    {
+      id: 3,
+      title: 'Node.js',
+      author: 'Sebastian Springer',
+    },
+    {
+      id: 4,
+      title: 'Javascript for Web Developer',
+      author: 'Matt Frisbie',
+    },
+  ];
+  return (
+    <>
+      <div className="">
+        { books.map((book) => (
+          <p key={book.id}>
+            {book.title}
+            {' '}
+            written by
+            {' '}
+            {book.author}
+          </p>
+        )) }
```

Comment:
- [ ] Please create a `<Book />` component that takes the book data as props and renders the title and the author.
- [ ] Please ensure to add a button to delete the book. Kindly note that the button wouldn't remove the books yet; it is just a placeholder for now 👍

Reply:
Addressed